tmux/formater_test.go

```diff
@@ -355,24 +355,24 @@ func TestFlagsWithoutCount(t *testing.T) {
 					NumStaged:   3,
 				},
 			},
-			want: "StyleClear" + "StyleStagedSymbolStaged StyleModSymbolMod StyleStashSymbolStash",
+			want: "[style:clear]" + "[style:staged][symbol:staged] [style:modified][symbol:modified] [style:stash][symbol:stash]",
 		},
 		{
 			name: "all flags without counts",
 			styles: styles{
-				Clear:     "StyleClear",
-				Conflict:  "StyleConflict",
-				Modified:  "StyleMod",
-				Stashed:   "StyleStash",
-				Staged:    "StyleStaged",
-				Untracked: "StyleUntracked",
+				Clear:     "[style:clear]",
+				Conflict:  "[style:conflict]",
```

Comment:
just remove the use of `expandStyles` and simply pass the Config as-is
much simpler